internal/build/gen_index.sh

```
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -eu
+
+: ${GO:=go}
+
+MODULES=$(find * -name go.mod -exec dirname '{}' \;)
+GROUPS="pkg"
```

Shellcheck (0.10.0)

[warning] Variable was used as an array but is now assigned a string.

(SC2178)